test/unit/math/prim/scal/fun/grad_reg_lower_inc_gamma_test.cpp

```diff
@@ -0,0 +1,9624 @@
+    {2.7501, 4.2501, -0.1461998787653077},
+    {2.7501, 4.5001, -0.1291967440646857},
+    {2.7501, 4.7501, -0.1133640446055395},
+    {2.7501, 5.0001, -0.098841754600809},
+    {2.7501, 5.2501, -0.0856882589194021},
+    {2.7501, 5.5001, -0.07390155619592386},
+    {2.7501, 5.7501, -0.06343676958173178},
+    {2.7501, 6.0001, -0.05422021195493302},
+    {2.7501, 6.2501, -0.04616039288137851},
+    {2.7501, 6.5001, -0.03915640831353119},
+    {2.7501, 6.7501, -0.03310415430811722},
+    {2.7501, 7.0001, -0.02790077570462913},
+    {2.7501, 7.2501, -0.02344771485622144},
+    {2.7501, 7.5001, -0.01965267385428694},
+    {2.7501, 7.7501, -0.01643075219631114},
+    {2.7501, 8.0001, -0.01370497392192579},
+    {2.7501, 8.2501, -0.01140637560427952},
+    {2.7501, 8.5001, -0.00947378987028009},
+    {2.7501, 8.7501, -0.007853428307907295},
+    {2.7501, 9.0001, -0.006498342296245603},
+    {2.7501, 9.2501, -0.005367819867156723},
+    {2.7501, 9.5001, -0.004426760508404614},
+    {2.7501, 9.7501, -0.003645057180901676},
+    {2.7501, 10.0001, -0.002997005126727117},
+    {2.7501, 10.2501, -0.002460749735085233},
+    {2.7501, 10.5001, -0.002017780330159125},
+    {2.7501, 10.7501, -0.001652472841815628},
+    {2.7501, 11.0001, -0.001351681580641456},
+    {2.7501, 11.2501, -0.001104378488277884},
+    {2.7501, 11.5001, -0.000901337051867842},
+    {2.7501, 11.7501, -0.0007348573828028748},
+    {2.7501, 12.0001, -0.0005985286281203575},
+    {2.7501, 12.2501, -0.0004870248023165956},
+    {2.7501, 12.5001, -0.0003959302176656526},
+    {2.7501, 12.7501, -0.0003215908921066943},
+    {2.7501, 13.0001, -0.0002609885809379096},
+    {2.7501, 13.2501, -0.0002116343797904491},
+    {2.7501, 13.5001, -0.0001714791589666635},
+    {2.7501, 13.7501, -0.0001388383977058282},
+    {2.7501, 14.0001, -0.0001123292811515604},
+    {2.7501, 14.2501, -0.0000908181965594413},
+    {2.7501, 14.5001, -0.00007337701528366043},
+    {2.7501, 14.7501, -0.000059246772023019},
+    {2.7501, 15.0001, -0.00004780755278201844},
+    {2.7501, 15.2501, -0.00003855357901786629},
+    {2.7501, 15.5001, -0.00003107262906914791},
+    {2.7501, 15.7501, -0.00002502907107863935},
+    {2.7501, 16.0001, -0.00002014989624679314},
+    {2.7501, 16.2501, -0.00001621323941006686},
+    {2.7501, 16.5001, -0.00001303895758531739},
+    {2.7501, 16.7501, -0.00001048090809267683},
+    {2.7501, 17.0001, -8.42062784971344e-6},
+    {2.7501, 17.2501, -6.762165941594342e-6},
+    {2.7501, 17.5001, -5.427863972855604e-6},
+    {2.7501, 17.7501, -4.354914193641331e-6},
+    {2.7501, 18.0001, -3.492555013681393e-6},
+    {2.7501, 18.2501, -2.799788178946526e-6},
+    {2.7501, 18.5001, -2.243522371462155e-6},
+    {2.7501, 18.7501, -1.797064972792085e-6},
+    {2.7501, 19.0001, -1.438897777889726e-6},
+    {2.7501, 19.2501, -1.151684043567706e-6},
+    {2.7501, 19.5001, -9.21463814828373e-7},
+    {2.7501, 19.7501, -7.37002338121455e-7},
+    {2.7501, 20.0001, -5.8926283313
```

Comment:
I'm guessing being close to 0 does weird things. Can we just split this out and then have this for loop go from 1 to `n_tests` and not have this if inside?

Or better yet, remove the first element and test that explicitly once and then just loop over n_tests?